common/config.go

```diff
@@ -204,6 +204,14 @@ type Config struct {
 	// default is 120s
 	HTTPESSClientTimeout int `env:"HTTPESSClientTimeout"`
 
+	// HTTPESSObjClientTimeout is to specify the http client timeout for downloading models (or objects) in seconds for ESS
+	// default is 600s
+	HTTPESSObjClientTimeout int `env:"HTTPObjESSClientTimeout"`
+
+	// HTTPESSObjDownloadConcurrencyMultiplier specifies a number to multiple the number of threads by to set allowed concurrent downloads per CSS
+	// default is 1
+	HTTPESSObjDownloadConcurrencyMultiplier int `env:"HTTPESSObjDownloadConcurrencyMultiplier"`
```

Comment:
nit, this should probably be named for the CSS, instead of ESS.

Reply:
Updated code with the change